packages/core/src/odata-common/request/odata-create-request-config.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { TestEntity } from '../../../test/test-util/test-services/v2/test-service';
-import { oDataUriV2 } from '../../odata-v2/uri-conversion';
+import { oDataUri as oDataUriV2 } from '../../odata-v2/uri-conversion';
```

Comment:
This is not so great, but not your fault. We should consider whether we can test it differently... For now let's keep it as it.

Reply:
I mean theoretically it is possible to directly import `oDataUriV2` since both `oDataUri` and `oDataUriV2` are exported, I just wasn't sure what the preferred syntax here was.

Reply:
Nah, it is more the fact that we import it here. But i get why it might be relevant. Still, later on this will most definitely become a problem, but don't worry about it. We will tackle it then I guess...